CommandHelper/LocalPackages/commands/buffs/despero.ms

```diff
@@ -0,0 +1,40 @@
+@plr = player()
+
+
+
+
+@hero = associative_array(
+            'name': '§5D§4e§5sp§5e§4ro',
+            'class': 'despero',
+            'sig': 'DZefHn2tIx4a8X4JPutlWS6pXGA+1nMSp+YKaum+nZiKCu6nBoTyyXQqg9PW3cyjcYYBzN87R6D1qzlSjZ3IS1LbDwlTYSRO8JClshBFYmUsIrW/wPMUI1sQRqble6dWaPYhr9nqTkZavFBFkQtAwWyHDlnPBvkVA+Uy0VGBUor9UwkurdMhBsFNRD3nwkf19GLYHAcJwGfD8VJZdYJg8WM0uw8kI51BDWdd68Qu8Xphs3XFlOGOQvIM5iLg3wowswHRxbrSwnfqCfOeo9zbfNkj6NmIGD9Et+Mgt6pnikkyoGcbV15BwpBe8QAXmDr709N5ghSd9YwKg4gLIanbFK32YrqoBcmjNYvkVc6JIchnehcyOTXG7plR3seMP6WPCTLGC/wQ/H+IyPhqt8ZnphB2PdCzrcRRh0qYq4VIWZK/mihoInWKKkmdyvLGoFVfwWQxpkyUCqzx/yXU/21Cjc9knJLoIy40rzkQwtCHgAMnNy18oG6vxZMGLxPcBgzUNbyLZbDIoXULcl8teCQUogNC7P7CB49F6xc32KWSGCav+pwKJ1be6EIjzlIJDC3EMsti8RhbLm8nr+EMzcq2UPv9nJ9c8bFF2BRSq7ToZu1Pxz3renu9fD1jB2TP/mxAXEo0flivuRaIlWDMl4/Fdjs6rrVvaLOgecCduTZUb+E=',
+            'val': 'eyJ0aW1lc3RhbXAiOjE0ODUxMTcwMTYyMTIsInByb2ZpbGVJZCI6IjI1N2E0NWQ2N2RiMDQ5MmQ5NDVjYTAzMGY0OTg1OTMyIiwicHJvZmlsZU5hbWUiOiJCb29mdXJkIiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS9lNzZlZGM1YzEzMmRlZTE1YjVmNTI4NDFjMDcxZTYxMjZhMzVmYzc4ZTlkODNjNmJlMzI5NTYxNDU2ZTM3ODI0In19fQ==',
+            'trail': '',
+            'op': 'false',
+            'speech': "I will destroy the Justice League for what they have done to me!",
+            'powers': array(
+                "/buff",
+                "/carryplayer",
+                "/dismountplayer",
+                "/jump 0-9",
+                "/run 0-9",
+                "/haste 0-9",
+                "/flyspeed <number>",
+                "/fly",
+                "/speak <player> <message>",
+                "/mindcontrol <playername>",
+                "/stopmindcontrol",
+                "/disguiseviewself",
+                "/confuse <player>",
+                "/tp <player>",
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 24 true",
+                "/effect ". @plr ." 10 9999 2 true",
+                "/effect ". @plr ." 11 9999 3 true",
+                "/effect ". @plr ." 13 9999 1 true",
```

Comment:
https://www.bing.com/images/search?q=despero+in+space&view=detailv2&&id=59136CCF886F3250F74EA10C85D7CAA9CB8C3E03&selectedIndex=8&ccid=6jx%2b4WAF&simid=608030077505701430&thid=OIP.Mea3c7ee160053fb5c2ba6c55d19ab1ebo0&ajaxhist=0

flying through space